Podfile

```diff
@@ -9,16 +9,20 @@ target 'RInAppMessaging_Example' do
   pod 'RSDKUtils', '~> 2.1', :testspecs => ['Nimble', 'TestHelpers']
 
   abstract_target 'Tests-Common' do
-    pod 'Quick'
+    pod 'Quick', '~> 5.0'
     pod 'Nimble'
 
     target 'Tests'
 
     target 'UITests' do
-      # Version 6.1 currently relies on swift-nio version (2.38) which has following issue
+      # Shock 6.1 currently relies on swift-nio version (2.38) which has following issue:
       # https://github.com/apple/swift-nio/issues/2073
-      # Version 6.0 uses older swift-nio version which doesn't have that problem
-      pod 'Shock', '~> 6.0.0'
+      # The issue has been fixed in https://github.com/apple/swift-nio/pull/2082
+      # The fixed script has been used to generate custom SwiftNIOPosix podspec as a workaround
+      # This workaround can probably be removed after the next SwiftNIO release
+      # (Older versions of Shock cannot be used with Xcode 13.3)
+      pod 'SwiftNIOPosix', :podspec => 'External Podspecs/SwiftNIOPosix.podspec'
```

Comment:
I had to put this podspec in a separate folder because `pod lib lint` tried to evaluate it when it was in root dir.